bitnami/redis/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 18.8.1
+version: 18.8.2
```

Coul you update the version?
```suggestion
version: 18.8.3
```